src/integration-test/java/uk/gov/hmcts/juror/api/moj/report/bespoke/YieldPerformanceReportITest.java

```diff
@@ -80,25 +86,35 @@ void viewByCourt() {
         testBuilder()
             .payload(payload)
             .triggerValid()
-            .responseConsumer(this::verifyHeadingsCourt)
-            .responseConsumer(this::verifyCourtPayload);
+            .responseConsumer(this::assertHeadingsCourt)
+            .responseConsumer(this::verifyCourtPayloadNegativeBalance);
 
     }
 
     @Test
-    void viewByCourts() {
+    void viewByCourtPositiveBalance() {
 
         YieldPerformanceReportRequest payload =  YieldPerformanceReportRequest.builder()
             .allCourts(false)
-            .courtLocCodes(List.of("415","774"))
+            .courtLocCodes(List.of("417"))
             .fromDate(LocalDate.parse("2024-08-01"))
             .toDate(LocalDate.parse("2024-08-20"))
             .build();
```

Comment:
Should use getValidPayload() and override what you are changing

Reply:
updated